bom/pom.xml

```diff
@@ -27,7 +27,7 @@
         <infinispanVersion>8.0.2.Final</infinispanVersion>
         <mongodbVersion>3.0.3</mongodbVersion>
         <fongodbVersion>2.0.2</fongodbVersion>
-        <orientdbVersion>2.1.9</orientdbVersion>
+        <orientdbVersion>[2.1.12,)</orientdbVersion>
```

Comment:
As a reminder, we don't use range, just leave 2.1.12
And if you can put it in a sperate commit I can start include something